integration-tests/src/main/java/org/apache/druid/testing/utils/KinesisAdminClient.java

```diff
@@ -119,13 +126,14 @@ public void updatePartitionCount(String streamName, int newShardCount, boolean b
       // Wait until the resharding started (or finished)
       ITRetryUtil.retryUntil(
           () -> {
-            StreamDescription streamDescription = getStreamDescription(streamName);
-            int updatedShardCount = getStreamShardCount(streamDescription);
-            return verifyStreamStatus(streamDescription, StreamStatus.UPDATING) ||
-                (verifyStreamStatus(streamDescription, StreamStatus.ACTIVE) && updatedShardCount > originalShardCount);
+            String streamStatus = getStreamStatus(streamName);
+            int updatedShardCount = getStreamPartitionCount(streamName);
+            return (streamStatus.equals(StreamStatus.UPDATING.toString()) ||
+                    streamStatus.equals(StreamStatus.ACTIVE.toString()))
```

Comment:
It seems that the position of the braces has changed.
Please confirm if this is intentional.

Original is: `stream is updating OR (stream is active and updated count > orig count)`
New is: `(stream is updating OR stream is active) and updated count > orig count`

Reply:
Yes, it is intentional to account for the scenarios where the stream itself might have been in updating state due to some other operation